content/blog/2015-08-xx-service-discovery-etc.md

```diff
@@ -0,0 +1,318 @@
+---
+title: Custom service discovery with etcd
+created_at: 2015-06-23
+kind: article
+author_name: Fabian Reinartz 
+---
+
+In a [previous post](/blog/2015-06-01-advanced-service-discovery.md) we
```

Comment:
nanoc doesn't look at the links here and update them to the routes they will end up with in the website, so this link indeed doesn't work when running it locally. It needs to be `/blog/2015/06/01/advanced-service-discovery/`.